app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,28 @@
+class ExpensesController < ApplicationController
+  def index
+    @expense = Expense.all
+  end
+
+  def new
+    @expense = Expense.new
+  end
+
+  def create
+    @expense = Expense.new(expense_params)
+    @expense.build_balance(value: 1)
```

Comment:
Value shouldn't be hardcoded - you need to pass this through params :)

Reply:
Handle it later